bpfman-api/src/bin/rpc/rpc.rs

```diff
@@ -91,6 +92,21 @@ impl Bpfman for BpfmanLoader {
                     .map_err(|e| Status::aborted(format!("failed to create tcprogram: {e}")))?,
                 )
             }
+            Info::TcxAttachInfo(TcxAttachInfo {
+                priority,
+                iface,
+                position: _,
```

Comment:
can u drop `position` if its not in use ?

Reply:
`position` is set later when the program is attached.